File: crates/once-frontend/prelude/examples/android-binary-minimal/apps/hello/src/dev/once/hello/MainActivity.java
package dev.once.hello;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public final class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView label = new TextView(this);
        label.setText(getString(R.string.app_name));
        setContentView(label);
    }
}

File: crates/once-frontend/prelude/examples/android-binary-minimal/apps/hello/src/dev/once/hello/MainActivity.kt
package dev.once.hello

import android.app.Activity
import android.os.Bundle
import android.widget.TextView
import dev.once.greeting.Greeting

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val label = TextView(this)
        label.text = Greeting.message(getString(R.string.app_name))
        setContentView(label)
    }
}
